src/main/java/seedu/address/model/CourseList.java

```diff
@@ -0,0 +1,131 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.course.Course;
+import seedu.address.model.course.UniqueCourseList;
+
+/**
+ * Wraps all data at the address-book level
+ * Duplicates are not allowed (by .isSamePerson comparison)
```

Comment:
There are some instances where "address book" is still used instead of "CodeSphere" and "person" is used instead of "student". Maybe review the all files changed once more to fix the naming issues?